Documentation/installing-dependencies.rst

```diff
@@ -1,216 +1,293 @@
-  export CMAKE_PREFIX_PATH=$HOME
-  export EDITOR=vi
-  export CPATH=$HOME/include:$CPATH 
+  # write the path here which you created your directory in (you can figure it out by typing `pwd`)
+  export SEISSOL_BASE=$HOME/seissol
+
+  export PREFIX=$SEISSOL_BASE/local
+  export PATH=$PREFIX/bin:$PATH
+  export LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib64:$LIBRARY_PATH
+  export LD_LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib64:$LD_LIBRARY_PATH
+  export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig:$PREFIX/lib64/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$PREFIX:$CMAKE_PREFIX_PATH
+  export CMAKE_INSTALL_PREFIX=$PREFIX
+  export CPATH=$PREFIX/include:$CPATH 
+  export C_INCLUDE_PATH=$PREFIX/include:$C_INCLUDE_PATH
+  export CXX_INCLUDE_PATH=$PREFIX/include:$CXX_INCLUDE_PATH
+  export EDITOR=nano # or e.g. vi,vim
+
+  # run "source ~/seissol/setup.sh" to apply environment to the current shell
 
-  # run "exec bash" or "source ~/.bashrc" to apply environment to the current shell
+Required Dependencies
+~~~~~~~~~~~~~~~~~~~~~
 
-Installing CMake
-~~~~~~~~~~~~~~~~
+We assume that you have a compiler already installed.
+The same goes for a suitable Python installation.
+You will also need CMake in version 3.20.0 or above. Most likely, you system will already have a
+version of CMake installed; however, you may have to load a module to get a new enough version.
+
+If you do not have CMake in a new enough version available, you may also install it manually as follows.
 
 .. code-block:: bash
 
   # you will need at least version 3.20.0 for GNU Compiler Collection 
   (cd $(mktemp -d) && wget -qO- https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-Linux-x86_64.tar.gz | tar -xvz -C "." && mv "./cmake-3.20.0-linux-x86_64" "${HOME}/bin/cmake")
   
-  # use version 3.16.2 for Intel Compiler Collection
-  (cd $(mktemp -d) && wget -qO- https://github.com/Kitware/CMake/releases/download/v3.16.2/cmake-3.16.2-Linux-x86_64.tar.gz | tar -xvz -C "." && mv "./cmake-3.16.2-Linux-x86_64" "${HOME}/bin/cmake")
-  
-  ln -s ${HOME}/bin/cmake/bin/cmake ${HOME}/bin
+  ln -s ${PREFIX}/bin/cmake/bin/cmake ${PREFIX}/bin
 
-Note that this extracts CMake to the directory ${HOME}/bin/cmake, if you wish you can adjust that path.
+Note that this extracts CMake to the directory ``${PREFIX}/bin/cmake``, if you wish you can adjust that path. Note that you may now also use ``ccmake`` to get a terminal UI for configuring the following libraries.
   
+Required Libraries
+~~~~~~~~~~~~~~~~~~
+
+The following libraries need to installed for all SeisSol CPU and GPU builds.
+To get a working CPU build, installing all libraries described here is enough.
+However, installing a GEMM generator and a graph partitioner is still recommended for better performance and better load balancing, respectively.
+
 Installing HDF5
-~~~~~~~~~~~~~~~
+"""""""""""""""
+
+We begin with HDF5 which is needed for reading meshes in PUML format (the default format in SeisSol, and the output of PUMgen).
```

Comment:
`HDF5, which`

Reply:
I would remove "We begin with".
HDF5 is needed ...